package/main/daml/Daml.Finance.Holding/daml.yaml

```diff
@@ -4,15 +4,15 @@
 sdk-version: 2.7.0
 name: daml-finance-holding
 source: daml
-version: 2.0.1
```

Comment:
Do we want to merge the version bumps in main? No strong opinion from my side

Reply:
First I thought no, but then I realized that if we need to do another snapshot release before `2.8` these version bumps would be a good starting point (since packages that did not change since the last snapshot would not be snapshot-released again).